test/sagemaker_tests/pytorch/training/requirements.txt

```diff
@@ -1,22 +1,19 @@
 botocore>1.0,<2.0
 boto3>1.0,<2.0
-awscli<2
+awscli>=1.27.51
 protobuf>=3.20,<3.21
 sagemaker
-smdebug>=1
 coverage==4.5.3
 docker-compose
 flake8==3.7.7
 Flask==1.1.1
 mock==2.0.0
-pytest==4.4.0
-pytest-cov==2.7.1
-pytest-rerunfailures==8.0
+pytest
+pytest-cov
+pytest-rerunfailures
 pytest-xdist==1.28.0
 PyYAML==5.4
 requests==2.24
```

Comment:
we do not need torch/torchvision?

Reply:
Since this is the test driver, I think we wouldn't need to install torch/torchvision outside of DLC?
AWS PyTorch and torchvision have been installed in the DLC.